jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```
@@ -76,11 +77,8 @@
 
     static
     {
-        for (Authenticator.Factory factory : ServiceLoader.load(Authenticator.Factory.class))
-        {
-            __knownAuthenticatorFactories.add(factory);
-        }
-
+        TypeUtil.serviceStream(ServiceLoader.load(Authenticator.Factory.class))
+            .forEach(__knownAuthenticatorFactories::add);
```

This is a different behavior. Before we were throwing if there was a problem finding an `Authenticator.Factory`, but now we ignore the exception and if we manage to start we won't work and we don't know why.

---

Discovering `AuthenticatorFactory`s with the `ServiceLoader` is a very recent change so I think this is okay. It will also allow jetty-jaspi to work in jetty-all.